package.json

```diff
@@ -17,13 +17,13 @@
     "@polkadot/types-known": "5.6.1",
     "@polkadot/util": "7.2.1",
     "@polkadot/rpc-core": "5.6.1",
```

Comment:
should update all polkadot resolutions to match SDK's, or try remove them

Reply:
Prob thats why build is failing, will check this out thanks

Reply:
yeah the "/sr25519" not exported error is related to it pulling an older polkadot version